packages/metal-dom/src/domNamed.js

```diff
@@ -448,6 +449,11 @@ export function on(element, eventName, callback, opt_capture) {
 	if (isString(element)) {
 		return delegate(document, eventName, element, callback);
 	}
+	var keyConfig = getKeyboardEventConfig(eventName);
+	if (keyConfig) {
```

Comment:
This `if` clause has the same logic as the one for `customConfig` below. We could take advantage of this by reusing the `customConfig` variable, like this:

```js
var customConfig = customEvents[eventName] || getKeyboardEventConfig(eventName);
```

That way we can remove all these extra lines :)